tools/src/test/resources/1.7/informal-invalid-component-versionRange-non-extraneous-explicit.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<bom xmlns="http://cyclonedx.org/schema/bom/1.7"
+     serialNumber="urn:uuid:3e671687-395b-41f5-a30f-a58921a69b79" version="1"
+>
+    <!--
+    this would be formal, if the support for XSD1.1's `assert` was properly implemented
+    in validators and tools digesting XML.
+    -->
+    <components>
+        <component type="library" isExtraneous="false">
+            <name>InvalidVersions</name>
+            <versionRange><![CDATA[>=9.0.0|<10.0.0]]></versionRange>
```

Comment:
I just took the version-part of VERS.
maybe this was wrong, need to read the specs of vers again ...

Reply:
fixed